client-linuxapp/src/serviceinfo.rs

```diff
@@ -626,6 +645,8 @@ async fn process_serviceinfo_in(si_in: &ServiceInfo, si_out: &mut ServiceInfo) -
         create_user(sshkey_user.as_ref().unwrap()).context("Error creating user")?;
         install_ssh_key(sshkey_user.as_ref().unwrap(), sshkey_key.as_ref().unwrap())
             .context("Error installing SSH key")?;
+        set_passwordless_login(sshkey_user.as_ref().unwrap())
+            .context("Error setting up passwordless login")?;
```

Comment:
hold on - create another PR for this, this doesn't belong here as here we're just fixing the user creation

Reply:
make sense split into #306